packages/vuetify/src/components/VDataTable/VDataTableRow.tsx

```diff
@@ -136,10 +141,24 @@ export const VDataTableRow = genericComponent<new <T>(
             >
               {{
                 default: () => {
-                  if (slots[slotName] && !mobile.value) return slots[slotName]?.(slotProps)
-
                   if (column.key === 'data-table-select') {
-                    return slots['item.data-table-select']?.(slotProps) ?? (
+                    return slots['item.data-table-select'] ? (
+                      <VDefaultsProvider
+                        defaults={{
+                          VCheckboxBtn: {
+                            disabled: !item.selectable,
+                            modelValue: isSelected([item]),
```

Comment:
Is there any point having these provided if you have to use `v-bind="props"` anyway to make the component work?